ajax/getUsers.php

```diff
@@ -1,11 +1,13 @@
 <?php
-OCP\User::checkLoggedIn ();
-OCP\JSON::callCheck ();
+
+OCP\User::checkLoggedIn();
+OCP\JSON::callCheck();
+
+header('Content-Type: application/json; charset=utf-8');
 
 $limit = 10;
 $offset = 0;
 
-$users = OCP\User::getDisplayNames((string)$_GET['search'], $limit, $offset);
+$users = OCP\User::getDisplayNames((string) $_GET['search'], $limit, $offset);
 
-OCP\JSON::encodedPrint ( $users );
-?>
+json_encode($users);
```

Comment:
:laughing: you are right. It was late yesterday... But the header should be ok, see line 6.